packages/grafana-ui/src/components/PanelMenu/PanelMenuItem.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2, PanelMenuItem } from '@grafana/data';
+import { Icon } from '../Icon/Icon';
+import { IconName } from '../../types';
+import { useStyles2 } from '../../themes';
+
+interface Props {
+  item: PanelMenuItem;
+  className?: string;
+  children?: React.ReactNode;
+}
+
+export const PanelMenuListItem = ({ item, className, children }: Props) => {
+  const styles = useStyles2(panelMenuItemStyles);
+  return (
+    <li className={styles.subMenuItemList} role="none">
+      <button
+        onClick={item.onClick}
+        role="menuitem"
+        tabIndex={-1}
+        className={styles.menuItem}
+        aria-keyshortcuts={item.shortcut}
```

Comment:
Currently the `shortcut` values passed here don't conform to the spec, so I think they need to be processed before getting passed to `aria-keyshortcuts`.